packages/tracing/src/browser/metrics.ts

```diff
@@ -79,14 +79,14 @@ export class MetricsInstrumentation {
 
             if (entry.name === 'first-paint' && shouldRecord) {
               IS_DEBUG_BUILD && logger.log('[Measurements] Adding FP');
-              this._measurements['fp'] = { value: entry.startTime };
-              this._measurements['mark.fp'] = { value: startTimestamp };
+              this._measurements['fp'] = { value: entry.startTime, unit: 'millisecond' };
```

Comment:
wait the units are supposed to be send in full form? We can't use `ms` or `s`?

Reply:
@jan-auer informed me that fully written out and singular is the way to go

@jan-auer can you confirm?

Reply:
This is the source of truth: https://github.com/getsentry/relay/pull/1217

So we should use the fully written out version.